app/src/main/java/com/rage/clamber/Adapters/ClimbsRecyclerViewAdapter.java

```diff
@@ -0,0 +1,85 @@
+package com.rage.clamber.Adapters;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.rage.clamber.R;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+
+/**
+ * Adapter to display the rows of climbs.
+ */
+public class ClimbsRecyclerViewAdapter extends RecyclerView.Adapter<ClimbsRecyclerViewAdapter.ClimbsViewHolder> {
+
+    //TODO: Set up a listener interface
+
+    String [] climbGrades;
```

Comment:
you'll (almost) always want to have some sort of access modifier, probably just `protected` here